sessions/testing.qmd

```diff
@@ -15,18 +15,21 @@
 
 **Session objectives**:
 
+-   Gain an understanding of the usefulness and importance of 
+    formal testing
```

Comment:
Nice first start! As I mentioned to @rmj822 in #15, you want the objective to be specific and just as much on the "why/what" as the how.

So in this case, an edit would be:

```suggestion
-   Explain what "testing" means when developing packages, describe how they are structured, and construct simple tests of functions. 
```

In this case, we're saying that learners will remember/recall something, gain understanding on what tests means, and to create/apply this knowledge as a skill to the problem.